numpy/random/tests/test_generator_mt19937.py

```diff
@@ -1259,6 +1270,18 @@ def test_multivariate_normal(self, method):
         assert_raises(ValueError, random.multivariate_normal,
                       mu, np.eye(3))
 
+    @pytest.mark.parametrize("method", ["svd", "eigh", "cholesky"])
+    def test_multivariate_normal_basic_stats(self, method):
+        n_s = 1000
+        mean = np.array([1, 2])
+        cov = np.array([[2, 1], [1, 2]])
+        s = random.multivariate_normal(mean, cov, size=(n_s,), method=method)
+        s_center = s - mean
+        cov_emp = (s_center.T @ s_center) / (n_s - 1)
+        # these are pretty loose and are only designed to detect major errors
+        assert np.all(np.abs(s_center.mean(-2)) < 0.2)
+        assert np.all(np.abs(cov_emp - cov) < 0.4)
```

Comment:
This could be as tight as before if it works with the seeded generator it should be repeatable.